_pylibczi/IndexMap.cpp

```diff
@@ -35,10 +35,10 @@ namespace pylibczi {
   IndexMap::operator<(const IndexMap& other_)
   {
 
-      auto match = std::find_if(s_sortOrder.begin(), s_sortOrder.end(), [&](const libCZI::DimensionIndex& dim) {
+      auto match = std::find_if(s_sortOrder.begin(), s_sortOrder.end(), [&](const libCZI::DimensionIndex& dim_) {
 
-          auto matchDim = [dim](const MapType::value_type& p) -> bool {
-              return (p.first==dim);
+          auto matchDim = [dim_](const MapType::value_type& p_) -> bool {
```

Comment:
For my education, why append an `_`?

Reply:
It's part of the Mozilla naming convention I think. Parameters to functions are given a trailing _ symbol. It's to visually separate them from local scoped parameters. Also, there's more diversity in naming conventions in C++ it's not a global thing. I intend to add a .clang-tidy file so that the rules I'm using are shared across the project but I haven't gotten to it yet.